.github/workflows/builder_go_slsa3.yml

```diff
@@ -22,6 +22,13 @@ env:
   GENERATED_BINARY_NAME: compiled-binary
   # Builder
   BUILDER_BINARY: builder
+  BUILDER_RELEASE_BINARY: slsa-builder-go-linux-amd64
+  BUILDER_REPOSITORY: slsa-framework/slsa-github-generator
+  # Verifier
+  VERIFIER_REPOSITORY: slsa-framework/slsa-verifier
+  VERIFIER_RELEASE_BINARY: slsa-verifier-linux-amd64
+  VERIFIER_RELEASE_BINARY_SHA256: fb743bc6bb56908d590da66bfe5c266d003aa226b30fcada5f7b9e4aea43b52b
```

Comment:
Note: this hash is invalid. We will update it once we (the maintainers) have verified it's valid - see https://github.com/slsa-framework/slsa-github-generator/issues/74#issuecomment-1135312371